### PR TITLE
Feature/android build test

### PR DESCRIPTION
**Pull request by vithun4**
This pull request contains the new updated UI that was created to be a dashboard/ Digital Gauge Cluster for Pickerel. This was done using qml & c++ to modify and optimize the display for our needs.

### CODE REVIEW

File: src/FlightDisplay/FlyView.qml [hunk 1]
```diff
@@ -102,15 +102,6 @@ Item {
         guidedValueSlider:     _guidedValueSlider
     }
 
-    /*GuidedActionConfirm {
```
Comment (jmailloux) [inline src/FlightDisplay/FlyView.qml:105]: Looks like this was commented out.  Why delete it?  We should try to do as few changes as possible to what is there.

File: src/FlightDisplay/FlyView.qml [hunk 2]
```diff
@@ -134,15 +125,15 @@ Item {
         color:              qgcPal.window
         visible:            false
     }
-
+    /*
```
Comment (jmailloux) [inline src/FlightDisplay/FlyView.qml]: Can you add comments as to why this was commented out?

File: src/FlightDisplay/FlyViewWidgetLayer.qml [hunk 3]
```diff
@@ -49,233 +51,843 @@ Item {
...
     }
+
+    
+
+
+
+
+
+
+//     QGCToolInsets {
```
Comment (jmailloux) [inline src/FlightDisplay/FlyViewWidgetLayer.qml]: I guess all of this commented out code is what was there before?  Might as well get rid of it.  Don't think it serves any value.

File: src/api/QGCCorePlugin.cc [hunk 4]
```diff
@@ -293,16 +293,13 @@ void QGCCorePlugin::factValueGridCreateDefaultSettings(const QString& defaultSet
 {
     HorizontalFactValueGrid factValueGrid(defaultSettingsGroup);
 
-    bool        includeFWValues = factValueGrid.vehicleClass() == QGCMAVLink::VehicleClassFixedWing || factValueGrid.vehicleClass() == QGCMAVLink::VehicleClassVTOL || factValueGrid.vehicleClass() == QGCMAVLink::VehicleClassAirship;
+    //bool        includeFWValues = factValueGrid.vehicleClass() == QGCMAVLink::VehicleClassFixedWing || factValueGrid.vehicleClass() == QGCMAVLink::VehicleClassVTOL || factValueGrid.vehicleClass() == QGCMAVLink::VehicleClassAirship;
```
Comment (jmailloux) [inline src/api/QGCCorePlugin.cc]: Again why comment out vs just delete?

**Review by jmailloux — CHANGES_REQUESTED** [review]
Why so many changes in the mavlink submodule folder?  I thought we were going to just keep it to the updates we made.

File: src/FlightDisplay/FlyViewWidgetLayer.qml (hunk 3 above)
Comment (vithun4) [reply src/FlightDisplay/FlyViewWidgetLayer.qml]: Removed.

File: src/api/QGCCorePlugin.cc (hunk 4 above)
Comment (vithun4) [reply src/api/QGCCorePlugin.cc]: removed

File: src/FlightDisplay/FlyView.qml (hunk 2 above)
Comment (vithun4) [reply src/FlightDisplay/FlyView.qml]: removed

File: src/FlightDisplay/FlyView.qml (hunk 1 above)
Comment (vithun4) [reply src/FlightDisplay/FlyView.qml:105]: This was moved to another part of the code, this is the system arm slider which I had moved to the speed section of the UI.

**Review by jmailloux — APPROVED** [review]

**Pull request merged**